tests/parsing/GlobbingTest.py

```diff
@@ -211,10 +217,11 @@ def test_multiple_patterns(self):
         non_matches = ['aXbX', 'aXcX']
         self._test_fnmatch(pattern, matches, non_matches)
 
+    def test_empty_globs(self):
         pattern = []
         matches = ['anything', 'anything_else']
         non_matches = []
-        self._test_fnmatch(pattern, matches, non_matches)
+        self._test_fnmatch_without_globs(pattern, matches, non_matches)
```

Comment:
The test could just be:

```py
pattern = []
matches = []
non_matches = ['anything', 'anything else']

self._test_fnmatch(pattern, matches, non_matches)
```

Instead of a new function.